### PR TITLE
add WARC GPT world tour blog post

### PR DESCRIPTION
**Pull request by clare-stanton**

### CODE REVIEW

File: app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: "WARC-GPT \u201Con tour\u201D: Talk transcript and slide decks"
+author: matteo-cargnelutti
+excerpt_separator: <!--more-->
+tags:
+- AI Research
+---
+<img src="https://lil-blog-media.s3.amazonaws.com/warc-gpt-banner.png" alt=""/>
+
+Over the past couple of months, my colleague Kristi Mukk and I had the opportunity to talk about [WARC-GPT](https://lil.law.harvard.edu/blog/2024/02/12/warc-gpt-an-open-source-tool-for-exploring-web-archives-with-ai/) and the concept of Librarianship of AI to the greater [GLAM](https://en.wikipedia.org/wiki/GLAM_(cultural_heritage) community:
```
Comment (matteocargnelutti) [inline app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md]:
```suggestion
Over the past couple of months, my colleague Kristi Mukk and I had the opportunity to talk about [WARC-GPT](https://lil.law.harvard.edu/blog/2024/02/12/warc-gpt-an-open-source-tool-for-exploring-web-archives-with-ai/) and the concept of Librarianship of AI to the greater [GLAM](https://en.wikipedia.org/wiki/GLAM_(cultural_heritage)) community: 
```

File: app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
+title: "WARC-GPT \u201Con tour\u201D: Talk transcript and slide decks"
+author: matteo-cargnelutti
```
Comment (matteocargnelutti) [inline app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md]:
```suggestion
author: matteo-cargnelutti,kristi-mukk,clare-stanton
```

File: app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+excerpt_separator: <!--more-->
+tags:
+- AI Research
+---
+<img src="https://lil-blog-media.s3.amazonaws.com/warc-gpt-banner.png" alt=""/>
+
+Over the past couple of months, my colleague Kristi Mukk and I had the opportunity to talk about [WARC-GPT](https://lil.law.harvard.edu/blog/2024/02/12/warc-gpt-an-open-source-tool-for-exploring-web-archives-with-ai/) and the concept of Librarianship of AI to the greater [GLAM](https://en.wikipedia.org/wiki/GLAM_(cultural_heritage) community: 
+
+* In March, at an event organized by Harvard Library about [AI at Harvard Library](https://libcal.library.harvard.edu/event/12113771).
+* In April, at the [2024 Web Archiving Conference of the International Internet Preservation Consortium ](https://netpreserve.org/ga2024/)(IIPC), which was held in Paris.
```
Comment (bensteinberg) [inline app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md]:
```suggestion
* In April, at the [2024 Web Archiving Conference of the International Internet Preservation Consortium](https://netpreserve.org/ga2024/) (IIPC), which was held in Paris.
```

File: app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+You can view WARC-GPT’s source code and case study at [https://github.com/harvard-lil/warc-gpt](https://github.com/harvard-lil/warc-gpt). 
+
+This project is part of our [ongoing series exploring how artificial intelligence changes our relationship to knowledge](https://lil.law.harvard.edu/about/ai).
+
+---
+## Video
+Excerpt from the talk we gave at an event organized by the [Web Archiving section of the Society of American Archivists](https://connect.archivists.org/discussion/discussion-web-archiving-and-ai-1) on June 14th, 2024. 
+
+<div class="embed-container">
+  <iframe src="https://vimeo.com/964817363"
```
Comment (bensteinberg) [inline app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md]: This should be `<iframe src="https://player.vimeo.com/video/964817363?h=252032cc56" width="640" height="360" frameborder="0" allow="autoplay; fullscreen; picture-in-picture" allowfullscreen></iframe>` but I can't figure out how to make a multi-line suggestion. Let me know if I should just push the change.

File: app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+ <!--more-->
+
+You can view WARC-GPT’s source code and case study at [https://github.com/harvard-lil/warc-gpt](https://github.com/harvard-lil/warc-gpt). 
+
+This project is part of our [ongoing series exploring how artificial intelligence changes our relationship to knowledge](https://lil.law.harvard.edu/about/ai).
+
+---
+## Video
+Excerpt from the talk we gave at an event organized by the [Web Archiving section of the Society of American Archivists](https://connect.archivists.org/discussion/discussion-web-archiving-and-ai-1) on June 14th, 2024.
```
Comment (bensteinberg) [inline app/_posts/2024-06-28-warc-gpt-on-tour-talk-transcript-and-slide-decks.md]:
```suggestion
Here's an excerpt from the talk we gave at an event organized by the [Web Archiving section of the Society of American Archivists](https://connect.archivists.org/discussion/discussion-web-archiving-and-ai-1) on June 14th, 2024:
```

**Review by bensteinberg — CHANGES_REQUESTED** [review]
I've made a few suggestions, including a fix for the Vimeo embed. See also Matteo's changes. LMK if I should push any changes directly.

**Pull request merged**